docs/source/tutorial.rst

```diff
@@ -182,44 +244,212 @@ Sawtooth Lake includes additional transaction processors:
+processor written in Python.
+
+One of the on-chain settings is the list of supported transaction families. 
+To configure this setting, follow these steps:
+
+Step One: Start Config Family Processor
+---------------------------------------
+
+To start the config family transaction processor, run the following commands from the 
+Vagrant CLI:
+
+.. code-block:: console
+
+  $ tp_config tcp://localhost:40000
+
+Confirm that the transaction processor registers with the validator by viewing the Vagrant shell 
+in which the validator is running. A successful registration event produces the following output:
+
+.. code-block:: console
+  
+  [21:03:55.955 INFO    processor_handlers] registered transaction processor: identity=b'6d2d80275ae280ea', family=sawtooth_config, version=1.0, encoding=application/protobuf, namespaces=<google.protobuf.pyext._message.RepeatedScalarContainer object at 0x7e1ff042f6c0>
+  [21:03:55.956 DEBUG   interconnect] ServerThread sending TP_REGISTER_RESPONSE to b'6d2d80275ae280ea'
+
+
+Step Two: Create And Submit Batch
+----------------------------------
+
+In the example below, a JSON array is submitted to the `sawtooth config`
+command, which creates a batch of transactions containing the configuration
+change. The `intkey load` command is then used to submit the batch to the
+validator.
+
+The JSON array used tells the validator or validator network to accept transactions of the following types:
+
+* intkey
+* sawtooth_config
+
+To create and submit the batch containing the new configuration, enter the
+following commands from the Vagrant CLI:
 
 .. code-block:: console
 
-  $ intkey populate -o initial_state -P 100
-  $ intkey generate -o inc_dec_transactions -c 100
-  $ intkey load -f initial_state
-  $ intkey load -f inc_dec_transactions
+  $ sawtooth admin keygen my_key
+  $ sawtooth config set --key /home/ubuntu/sawtooth/etc/keys/my_key.wif sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0", "encoding": "application/protobuf"}, {"family":"sawtooth_config", "version":"1.0", "encoding":"application/protobuf"}]'
+  $ intkey load -f config.batch
+
+
+Viewing Blocks and State 
+========================
 
-You can monitor the activity of the validator as it processes the batches, and
-the activity of the transaction processor as it processes the transactions, by
-switching to the respective terminal windows. 
+You can view the blocks stored in the block-chain, and the nodes of the Markle
+tree, using the sawtooth CLI.
+
+.. note::
+  
+  The sawtooth CLI provides help for all subcommands. For example, to get help
+  for the `block` subcommand, enter the command `sawtooth block -h`.
+
+Log in to the Vagrant environment with the command `vagrant ssh` to run the commands below. 
+
+Viewing List Of Blocks
+----------------------
+
+Enter the command `sawtooth block list` to view the blocks stored by the state:
+
+.. code-block:: console
+
+  $ sawtooth block list
```

Comment:
You are right. I had the rest_api running already, from another part of the tutorial. We should not assume that it is still running.